From 87373f6c2b48619bb11e566e5eb16ea49d47beb7 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Fri, 19 Apr 2019 17:07:10 +0200
Subject: [PATCH] Link with libiconv if needed

Commit 2cc3a93dcf2703b3b418e0a99975f556354fb1b1 added an include to
iconv which can be provided by libiconv so search and link for it if
needed

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: https://github.com/linknx/linknx/pull/41]
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 22f242c..779fd6f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -15,6 +15,7 @@ AC_CHECK_HEADER(argp.h,,[AC_MSG_ERROR([argp_parse not found])])
 AC_SEARCH_LIBS(argp_parse,argp,,[AC_MSG_ERROR([argp_parse not found])])
 
 # Checks for libraries.
+AC_SEARCH_LIBS(libiconv_open,iconv)
 LIBCURL_CHECK_CONFIG([yes], [7.14.0])
 
 # Checks for header files.
-- 
2.20.1

